src/inference/src/dev/core_impl.cpp

```diff
@@ -461,64 +701,85 @@ std::vector<std::string> ov::CoreImpl::get_available_devices() const {
     return devices;
 }
 
-ov::RemoteContext ov::CoreImpl::create_context(const std::string& device_name, const AnyMap& args) const {
-    auto parsed = ov::parseDeviceNameIntoConfig(device_name, args);
+ov::RemoteContext ov::CoreImpl::create_context(const std::string& device_name, const AnyMap& params) const {
+    auto parsed = ov::parseDeviceNameIntoConfig(device_name, params);
     return get_plugin(parsed._deviceName).create_context(parsed._config);
 }
 
-ov::AnyMap ov::CoreImpl::get_supported_property(const std::string& device_name, const ov::AnyMap& config) const {
-    std::vector<std::string> supportedConfigKeys;
+ov::AnyMap ov::CoreImpl::get_supported_property(const std::string& full_device_name,
```

Comment:
Is there test cases for this function?

Reply:
It's used in GetCore()->GetSupportedConfig(), so it's tested by all current OV tests.